src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -58,7 +58,17 @@
  *   &lt;property name=&quot;fileExtensions&quot; value=&quot;properties&quot; /&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>
+ * Example:
+ * </p>
+ * <p>
+ * File name: main.properties
+ * </p>
+ * <pre>
+ * host=localhost // OK
+ * key=value1 // Violation - duplicate property 'key'
+ * key=value2 // Violation - duplicate property 'key'
```

Comment:
1. Please leave violation comment only on line where it is reported.
2. Violation -> violation
3. Exaplanation of violation should be with comma, e.g. `violation, here is explanation`